tests/routes/pro/get_offers_test.py

```diff
@@ -33,10 +33,10 @@ def should_filter_by_venue_when_user_is_admin_and_request_specific_venue_with_no
         stock = offers_factories.StockFactory(offer=offer_on_requested_venue)
         client = TestClient(app.test_client()).with_auth(email=admin.email)
         path = f"/offers?venueId={humanize(requested_venue.id)}"
-        select_and_count_offers_number_of_queries = 2
+        select_offers_number_of_queries = 1
```

Comment:
c'est un détail. 
Plutôt que d'avoir n variables avec des nom à rallonge,  j'aime bien la notation: 
```python
nb_queries = testing.AUTHENTICATION_QUERIES
nb_queries += 1 # select offers
with assert_num_queries(nb_queries):
  # ...
```

Reply:
```suggestion
        select_offers_nb_queries = 1
```
J'ai préféré raccourcir un peu le nom de la variable dans ce cas